doc/source/tools/cqlsh.rst

```diff
@@ -441,8 +441,7 @@ Options that are common to both ``COPY TO`` and ``COPY FROM``.
   The string literal format for boolean values.  Defaults to ``True,False``.
 
 ``NUMPROCESSES``
-  The number of child worker processes to create for ``COPY`` tasks.  Defaults to a max of 4 for ``COPY FROM`` and 16
-  for ``COPY TO``.  However, at most (num_cores - 1) processes will be created.
+  The number of child worker processes to create for ``COPY`` tasks.  Defaults to a max of 16 or (num_cores - 1) processes, whichever is smaller.
```

Comment:
@neshkeev this one seems to be forgotten in https://github.com/apache/cassandra/pull/2062

Reply:
Is this actually true? Could you please check?

Reply:
I see this in your patch 

````
`NUMPROCESSES`::
  The number of child worker processes to create for `COPY` tasks.
  Defaults to a max of 4 for `COPY FROM` and 16 for `COPY TO`. However,
  at most (num_cores - 1) processes will be created.
````

cqlsh.adoc like 505

Reply:
I think that the meaning of both versions is correct, it is just formulated differently. Do you see some actual difference because I can not ....

Reply:
@smiklosovic , my question was about whether or not the statement in the proposed changes is true.

I see that this PR proposes to set the max value for `NUMPROCESSES` to 16 regardless if it's for `COPY FROM` or `COPY TO`, in my patch (which I didn't change) it says that the max value is 4 for `COPY FROM` and 16 for `COPY TO`. So which one is correct?

Reply:
ah right :D I apologize.

I will try to check it and return back to you.

Reply:
@neshkeev the suggested change in this PR is correct. I do not see anything in the code which would set it by default to 4 for `COPY FROM` and 16 for `COPY TO`. It is always capped to 16 or num_cores - 1, whatever is smaller.

Reply:
@neshkeev it should be also changed in `conf/cqlshrc.sample`

Reply:
I pushed the changes to https://github.com/apache/cassandra/pull/2062: [cqlsh.adoc](https://github.com/apache/cassandra/pull/2062/files#diff-7b9caadf725c996c57188d3403035f79bdff83dc12dd510da5faf970d0ec9cd2R507), [conf/cqlshrc.sample](https://github.com/apache/cassandra/pull/2062/files#diff-f55a268d347d301121adb03b1d33958837ed984295b83fbe474dde2781fe809fR159)

Reply:
thanks